### PR TITLE
adding comparison chart to about page

### PR DESCRIPTION
**Pull request by ebrillhart**
cc/ @paulathevalley @eliseworthy 

adding the comparison chart for when to use formidablecharts v. victory to the about page. going to flesh out with a paragraph or so. this seemed like the best place for it to live for now, until a more robust guide for upgrading to victory from formidablecharts can be created down the road.

### CODE REVIEW

**Comment by paulathevalley** [comment]
Here’s what the comparison table looks like now, but I’d like copy review/approval from @boygirl @kenwheeler @eliseworthy 

screen shot 2016-11-02 at 3 47 01 pm

also resolves #4

File: src/screens/about/components/comparison.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import VLOGO from "victory-docs/static/logotype-hero.svg";
+
+class ComparisonChart extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>When should I use FormidableCharts versus Victory?</h2>
+        <p>We wanted to provide developers with multiple charting options for their projects. FormidableCharts is
+         quick and easy to implement with minimal muss and fuss around styling, theming, and composing components.
+         However, Victory is much more flexible and powerful, and allows users to develop pretty much any kind of
```
Comment (boygirl) [inline src/screens/about/components/comparison.js]: The word "powerful" is not appropriate for Victory.

File: src/screens/about/components/comparison.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              </thead>
+              <tbody>
+                <tr>
+                  <td className="Table-td--same">Native Support</td>
+                </tr>
+                <tr>
+                  <td className="Table-td--same">Animations</td>
+                </tr>
+                <tr>
+                  <td>Advanced Chart Types</td>
```
Comment (boygirl) [inline src/screens/about/components/comparison.js]: They have the same chart types... So maybe "Pre-Built Charts" vs. "Modular Chart Components"

Comment (boygirl) [reply src/screens/about/components/comparison.js]: "Automatic Interactive Events" vs. "Flexible Event System"
"Emphasis on Ease of Use" Vs. "Emphasis on Flexibility"

File: src/screens/about/components/comparison.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                </tr>
+                <tr>
+                  <td>Flexible Data Formats <br/>with Data Accessors</td>
+                </tr>
+                <tr>
+                  <td>Custom Themes and Styles</td>
```
Comment (boygirl) [inline src/screens/about/components/comparison.js:104]: Data, theme, style stuff is true for both projects, I believe

File: src/screens/about/components/comparison.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+class ComparisonChart extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>When should I use FormidableCharts versus Victory?</h2>
+        <p>We wanted to provide developers with multiple charting options for their projects. FormidableCharts is
+         quick and easy to implement with minimal muss and fuss around styling, theming, and composing components.
+         However, Victory is much more flexible and powerful, and allows users to develop pretty much any kind of
+         chart configuration that they could need. Here's a chart to help you figure out what's best for your
```
Comment (boygirl) [inline src/screens/about/components/comparison.js]: chart -> table

File: src/screens/about/components/comparison.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+import LOGO from "../../../../static/logotype.svg";
+import VLOGO from "victory-docs/static/logotype-hero.svg";
+
+class ComparisonChart extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>When should I use FormidableCharts versus Victory?</h2>
+        <p>We wanted to provide developers with multiple charting options for their projects. FormidableCharts is
+         quick and easy to implement with minimal muss and fuss around styling, theming, and composing components.
```
Comment (boygirl) [inline src/screens/about/components/comparison.js]: Dislike "muss and fuss".

File: src/screens/about/index.js [hunk 6]
```diff
@@ -26,7 +27,7 @@ class About extends React.Component {
             <iframe src="https://ghbtns.com/github-btn.html?user=formidablelabs&repo=formidable-charts&type=fork&count=true&size=large" frameBorder="0" scrolling="0" width="158px" height="30px"></iframe>
           </p>
           <p>
-            FormidableCharts is a library that provides quick, composed, charts out-of-the-box for React and other Javascript projects.
+            FormidableCharts is a library that provides quick, composed charts out-of-the-box for React and other Javascript projects.
```
Comment (boygirl) [inline src/screens/about/index.js]: Dislike "fiddling"

File: src/screens/home/index.js [hunk 7]
```diff
@@ -11,63 +11,72 @@ class Home extends React.Component {
...
+                  Get Started <Icon glyph="internal-link" />
+                </Link>
+              </div>
+              */}
+              <div className="u-paddingSm">
+                <div className="Grid Grid--gutters Grid--full medium-Grid--1of3">
+                  <div className="Grid-cell u-textCenter">
+                    <h3>Composed</h3>
+                    <p>
+                      Build the charts you need quickly without the hassle of fitting the pieces together yourself.
```
Comment (boygirl) [inline src/screens/home/index.js]: Dislike "hassle"

File: src/screens/about/components/comparison.js (hunk 5 above)
Comment (boygirl) [reply src/screens/about/components/comparison.js]: Focus on the positives of each. Maybe something more like "focus on the data with pre-built, easy to use charts" vs . "Use Victory to create fully custom charts that integrate seamlessly with the look and feel of your project."

**Review by boygirl — CHANGES_REQUESTED** [review]
I really like the comparison table. 

Language like "muss and fuss" and "fiddling" all needs to go away, IMO.

File: src/screens/about/components/comparison.js (hunk 3 above)
Comment (paulathevalley) [reply src/screens/about/components/comparison.js:104]: There’s a lot of copy around "built-in themes" and "spend less time styling" etc. ...is this accurate? Or, does the copy need to be modified?

**Comment by paulathevalley** [comment]
@boygirl Made some copy change requests! I still have some questions, but take a look. 

screen shot 2016-11-03 at 3 28 30 pm

Some questions: 
- There is some copy differentiating FormidableCharts by explaining how easy it is to use prebuilt themes and styling. Is there a way we could reflect that in the tables? 
- Both words "composed" and "prebuilt" (also, is it pre-built or prebuilt?) are being used interchangeably. Is that helpful or confusing? Should we pick one? 
- Ideally, one table will be longer than the other to show that one project is more fully featured, robust, advanced, etc. First, is this true? Does Victory have features that FormidableCharts does not have?

File: src/screens/about/components/comparison.js [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+          When should I use FormidableCharts versus Victory?
+        </h2>
+        <p>
+          We wanted to provide developers with multiple charting options for their projects.
+        </p>
+        <p>
+          FormidableCharts is quick and easy to implement! Focus on the data with pre-built charts.
+        </p>
+        <p>
+          Victory is much more flexible, allowing users to develop pretty much any kind of chart configuration. Create fully custom charts that integrate seamlessly with the look and feel of your project.
```
Comment (boygirl) [inline src/screens/about/components/comparison.js]: "Victory is much more flexible, allowing users to create fully custom charts that integrate seamlessly with the look and feel of a project."

I think "pretty much any kind of chart" is unnecessary.

**Comment by boygirl** [comment]
@paulathevalley 

I think "Data Accessors" can be dropped from the table. Maybe ditto "series-style data" and "flexible data formats". Both have about equivalently flexible ways of handling data, just with slightly different organization. I'm not sure this detail is worth calling out.

I sticking with prebuilt, readymade etc is less confusing for formidable charts since we emphasize composability for victory

Maybe

| FormidableCharts | Victory |
| --- | --- |
| Web and Native Charts | Web and Native Charts |
| Pre-built Charts | Modular Chart Components |
| Emphasizes ease of use | Emphasizes Flexibility |
| Automatic Animations | Configureable Animations |
| Automatic Interactive Events | Flexible Event System |
| Readymade themes | Custom themes and styles |

Victory has a few features that FormidableCharts doesn't have, like error bars and candlestick components, but for the most part you can create all the same chart types in FormidableCharts but with more limited options / a lot of stuff already configured for you. The main advanced feature of Victory is that it lets you write your own advanced features. I'm not 100% sure how to capture that idea in a table.

**Comment by paulathevalley** [comment]
@boygirl Ok! Updated: 

screen shot 2016-11-04 at 10 53 28 am

Having only one similarity is not as visually obvious what it is, but it’s okay to leave it if needed. Could we split it the one similarity into two rows: Web Charts, Native Charts (or Native Support)?

I think it’s worth making the Victory table longer to keep the "it’s more advanced!" takeaway at a glance. So, ideas...
Custom Feature Support? 
Custom Advanced Features? 
Custom Component Support? 
Custom Chart Components (to match Modular Chart Components language)? 
DIY Components 😛 

Here’s what I’m thinking:

screen shot 2016-11-04 at 10 58 27 am

What do you think? (know that either option is totally ok!)

**Comment by boygirl** [comment]
Option 2! "Custom Feature Support" is exactly what I was looking for.

**Review by boygirl — APPROVED** [review]

**Comment by boygirl** [comment]
@paulathevalley Is this table going to end up somewhere on the victory lander too?

**Comment by paulathevalley** [comment]
@boygirl Cool! Will do. 

After discussing with @eliseworthy, our thinking is that FormidableCharts is the entry point to Victory. I think of the typical workflow as a one-way street: FormidableCharts --> Victory. We’re thinking it is not likely for someone to start with Victory and go for something more limiting. I think of it like a video game: once you max out your achievements (aka push against the limits of FormidableCharts), you upgrade to the next level (Victory). Therefore, all the "transition" guides would be on FormidableCharts dox, including this table, since that's the starting point, and it does not make as much sense to duplicate them on Victory dox... as you’ve already upgraded by then.

However, the formidable-charts release and ad buy are changing things up a bit. We are planning to have a banner across the top of the Victory homepage that links to a blog post on formidable.com which describes/compares FormidableCharts and Victory (and I think other charting libraries too?). This blog post task is on @eliseworthy’s plate: I believe she might already be proofreading it? And it’s on my plate to create a visual mockup for it, but we haven’t finalized the wording of the banner yet. 

We can talk more about the banner/transition guides on Monday if that’s easier. I would definitely love to hear your thoughts on wording and this approach in general. 

For now, I’m going to merge!

**Pull request merged**